dist/index.html

```diff
@@ -125,7 +125,7 @@
                     <tr>
                         <th class="small">Machine name</th>
                         <td class="small">
-                            <a target="_blank" href="https://secure.pub.build.mozilla.org/builddata/reports/slave_health/slave.html?name={{ job.machine_name }}">{{ job.machine_name }}</a>
+                            <a target="_blank" href="{{ getMachineNameUrl(job.machine_name) }}">{{ job.machine_name }}</a>
```

Comment:
The files under `dist/` are automatically generated when we run `grunt build` (see http://treeherder.readthedocs.org/deployment.html#updating-the-ui-build-prior-to-deployment) so shouldn't be updated by hand.

If you make the changes to just the files under `ui/`, we'll run grunt build later for you :-)